source-control/git-guidance/git-lfs-and-vfs.md

```diff
@@ -0,0 +1,156 @@
+# Using Git LFS and VFS for Git introduction
+
+**Git LFS** and **VFS for Git** are solutions for using Git with (large) binary files and large source trees.
+
+## Git LFS
+
+Git is very good and keeping track of changes in text-based files like code. But it is not that good in tracking binary files. For instance, if you store a Photoshop image file (PSD) in a repository, with every change, the complete file is stored again in the history. This can make the history of the Git repo very large, which makes a clone of the repository more and more time consuming.
+
+A solution to work with these kind of files is using Git LFS (or Git Large File System). This is an extension to Git and must be installed seperately. And it can only be used with a repository platform that supports LFS. GitHub.com and Azure DevOps for instance are platforms that have support for LFS.
+
+The way it works in short, is that a placeholder file is stored in the repo with information for the LFS system. It looks something like this:
+
+```shell
+version https://git-lfs.github.com/spec/v1
+oid a747cfbbef63fc0a3f5ffca332ae486ee7bf77c1d1b9b2de02e261ef97d085fe
+size 4923023
+```
+
+The actual file is stored in a separate storage. This way Git will track changes in this placeholder file, not the large file. The combination of using Git and Git LFS will hide this from the developer though. You will just work with the repository and files as before.
+
+When working with these large files yourself, you'll still see the git history grown on your own machine, as git will still start tracking these large files locally. But when you clone the repo, the history is actually pretty small. So it's beneficial for others not working directly on the large files.
+
+### Benefits of Git LFS
+
+* Uses the end to end Git workflow for all files
+* Git LFS supports file locking to avoid conflicts for undiffable assets
+* Git LFS is fully supported in Azure DevOps Services
+
+### Limitations of Git LFS
```

Comment:
Are these really limitations or pros/cons?

Reply:
We should mention that LFS is a paid service.

Reply:
@cmayomsft LFS is not a paid service, as far as I know - but please tell me if I'm wrong

Reply:
@cmayomsft good point, adjusted pros/cons!